src/app/Http/Controllers/BigQuestionsController.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use App\BigQuestion;
+use App\Question;
+
+
+class BigQuestionsController extends Controller
+{
+    // 問題リスト表示用
+    public function list(Request $request) 
+    {
+        $user = Auth::user();
+        $questions = Question::all();
+
+
+        if (!empty($request->input('list-ids')))
+        {
+            $list = $request->input('list-ids');
+
+            $lists = explode(',', $list);
+
+            foreach($lists as $index => $id){
+                $big_question = BigQuestion::where('id', $id)->first();
+                $big_question->sortID = $index + 1;
+                $big_question->save();
+            }
+        } else { }
```

Comment:
else消し忘れですかね？
特に何もしないのであれば消すかわざと残しているのであればコメント書いたほうが良さそうです
```suggestion
        } 
```